migrations/20190328115506_add_profile_to_establishment_table.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('profile_to_establishment')
```

Comment:
I would perhaps use a less ambiguous name - this table is _only_ for asru users, perhaps asru_establishment?

Reply:
done.